dotcom-rendering/src/web/lib/dailyArticleCount.ts

```diff
@@ -36,12 +36,14 @@ export const getDailyArticleCount = (): DailyArticleHistory | undefined => {
 	}
 };
 
+export const getToday = (): number => Math.floor(Date.now() / 86400000);
```

Comment:
You can use underscores to make large numbers easier to parse!

```suggestion
export const getToday = (): number => Math.floor(Date.now() / 86_400_000);
```